vignettes/programming_strategy.Rmd

```diff
@@ -444,6 +446,15 @@ deprecate_stop("x.y.z", "fun_xxx()", "new_fun_xxx()")
 ### END DEPRECATION
 ```
 
+For the former case above, please pass any input arguments into a call to the new function so that it still runs and pushes users towards adopting the new function.
+
+```
+fun_xxx <- function(dataset, new_var) {
+  deprecate_warn("x.y.z", "fun_xxx()", "new_fun_xxx()")
+  **new_fun_xxx(dataset, new_var = new_var)**
+}
+```
```

Comment:
Please put `r` after the opening three backticks. That way the code will have syntax highlighting. Also please remove the `**` around `new_fun_xxx()`. Unfortunately you cannot bold text inside a code block using markdown. Using the HTML tag `<b>` (opening) and `</b>` might work. Please check by knitting locally and looking at the output.

Reply:
ok updated - the bold was a nice to have and not really needed in hindsight so left out now